README.md

```diff
@@ -341,15 +343,46 @@ If you are interested in being a contributor and want to get involved in develop
 
 In general, PRs are welcome. We follow the typical "fork-and-pull" Git workflow.
 
+
+
+
  1. **Fork** the repo on GitHub
  2. **Clone** the project to your own machine
- 3. **Commit** changes to your own branch
- 4. **Push** your work back up to your fork
- 5. Submit a **Pull Request** so that we can review your changes
+ 3. [**Setup pre-commit**](#developer-setup) to verify code style (optional, but recommended)
+ 4. **Commit** changes to your own branch
+ 5. **Push** your work back up to your fork
+ 6. Submit a **Pull Request** so that we can review your changes
+
 
 **NOTE:** Be sure to merge the latest changes from "upstream" before making a pull request!
 
 
+#### Developer Setup
```

Comment:
should we consider adding `make pre-commit/install` and `make pre-commit/init` targets that automatically run the commands below?

Reply:
@mcalhoun @goruha to revive this PR a little bit: I came here looking for this as I expected it in the build-harness. I definitely think it makes sense to add a new module/target for this as it's likely not just pre-commit that needs to be brew-installed: [If you look at the Mac install instructions in pre-commit-terraform](https://github.com/antonbabenko/pre-commit-terraform#macos) those hooks also require gawk, coreutils, and a couple others that we may or may not use in the future.